content/en/tracing/trace_collection/otel_instrumentation/dotnet.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Custom Instrumentation of .NET Applications with the OpenTelemetry API
+kind: documentation
+code_lang: dotnet
+type: multi-code-lang
+code_lang_weight: 80
+description: 'Instrument your .NET application with OTel API, to send traces to Datadog.'
+further_reading:
+    - link: 'tracing/glossary/'
+      tag: 'Documentation'
+      text: 'Explore your services, resources, and traces'
```

Comment:
Just noting that we have further reading links in the front matter, but we don't have the Further Reading section at the bottom of the article.